code/game/objects/items/miscellaneous.dm

```diff
@@ -63,6 +63,22 @@
 
 	new /obj/effect/DPtarget(get_turf(src), pod)
 
+//HoS gun beacon
+/obj/item/choice_beacon/hosgun
+	name = "Head of Security gun beacon"
+	desc = "Choose one of several experimental guns created by leading scientists at NT to put it through proper field testing."
+
+/obj/item/choice_beacon/hosgun/generate_display_names()
+	var/static/list/hosgun_item_list
```

Comment:
I don't see why this needs to be static. There should only ever be one HoS gun beacon so you can just store the list as a regular list in the type definiton.

Reply:
Fixed